api/lib/opentelemetry/trace/span.rb

```diff
@@ -74,8 +74,9 @@ def set_attribute(key, value)
       #   numeric type.
       #
       # @return [self] returns itself
-      def add_event(name, **attrs)
+      def add_event(name, attrs = nil)
```

Comment:
should we default to empty hash? `attrs = {}`. I've also seen `EMPTY_HASH = {}.freeze`, `def add_event(name, attrs = EMPTY_HASH)`

Reply:
The `attrs` is optional. I'd prefer to avoid the allocation if no attributes are provided.